pkg/controller/cassandradatacenter/statefulset.go

```diff
@@ -101,13 +100,16 @@ func newPodSpec(cdc *cassandraoperatorv1alpha1.CassandraDataCenter, volumes []co
 }
 
 func newCassandraContainer(cdc *cassandraoperatorv1alpha1.CassandraDataCenter, dataVolumeClaim *corev1.PersistentVolumeClaim, configVolume *corev1.Volume) *corev1.Container {
-	const OperatorConfigVolumeMountPath = "/tmp/operator-config"
+	const (
+		OperatorConfigVolumeMountPath = "/tmp/operator-config"
+		UserConfigVolumePath = "/etc/cassandra/cassandra.yaml.d/003-user-overrides.yaml"
```

Comment:
This should be `/tmp/user-config` (and renamed to `...MountPath`), since the user ConfigMap and VolumeSource can contain multiple files.

The entry point script will then take care of moving these all to underneath `/etc/cassandra`

Reply:
:+1: